recipe/meta.yaml

```diff
@@ -7,12 +7,13 @@ package:
 source:
   fn: qtpy-{{version}}.tar.gz
   url: https://pypi.io/packages/source/q/qtpy/QtPy-{{version}}.tar.gz
-  sha256: 3d20f010caa3b2c04835d6a2f66f8873b041bdaf7a76085c2a0d7890cdd65ea9
+  sha256: bbd61f8d6480a01cec39ad94249dbde7d0a8fce2aca61ff5037b645c4fd13e02
```

Comment:
there should be a new sha256 for `1.11.2` https://github.com/conda-forge/qtpy-feedstock/blob/7681e9d1466fe4580910fc9653faf5b2fddabdc2/recipe/meta.yaml#L10